unlock-app/src/components/interface/checkout/Connected.tsx

```diff
@@ -217,6 +219,7 @@ export function Connected({
           send('UNLOCK_ACCOUNT')
         }}
         authenticateWithProvider={authenticateWithProvider}
+        title="Have a crypto wallet?"
```

Comment:
```suggestion
        title={title}
```
we can use title also here?

Reply:
No there is no `title` in this component.